src/pcap/cmd/pcap-api/config.go

```diff
@@ -26,19 +27,18 @@ var DefaultAPIConfig = APIConfig{
 	Agents: &pcap.AgentMTLS{
 		MTLS: nil,
 	},
-	DrainTimeout:       10 * time.Second,                                                                                               //nolint:gomnd // default value
-	ConcurrentCaptures: 5,                                                                                                              //nolint:gomnd // default value
-	ManualEndpoints:    pcap.ManualEndpoints{Targets: []pcap.AgentEndpoint{{IP: "localhost", Port: 8083, Identifier: "test-agent/1"}}}, //nolint:gomnd // default value
+	DrainTimeout:       10 * time.Second,
+	ConcurrentCaptures: 5,
 }
 
 type APIConfig struct {
-	cmd.CommonConfig
-	Agents             *pcap.AgentMTLS `yaml:"agents"`
-	ConcurrentCaptures int             `yaml:"concurrent_captures"`
-	DrainTimeout       time.Duration   `yaml:"drain_timeout"`
+	cmd.CommonConfig   `yaml:"common_config"` // fixme: workaround by a18e to be able to parse a config file again, may not be desired
```

Comment:
I don't like this whole contraption. I will work a bit on resolving this "common" `cmd` package and, the `CommonConfig` etc.